### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by RadTechDad** [keiwando/toonsquid#23] [Q&A] [answered]
Do you have a roadmap of planned features?

I love ToonSquid and I'm really wanting to get behind this 200%!

**Comment by keiwando** [comment, marked as answer]
Thank you so much!

I'm currently not planning on having a public roadmap since being able to very quickly reprioritize different features and not feeling like I need to hit some arbitrarily chosen public deadline allows me to overall work a lot more efficiently. 

I generally take note of every feature request and prioritize them based on how many times it gets requested (among many other factors). If you have any suggestions I'd love to hear them!

The next update (1.6) is going to contain the following features (and some more):
- a duplicate drawing button (in addition to "add" and "extend")
- more onion skin settings
- an option to disable finger drawing
- rasterization of animation layers into frame-by-frame drawings
- A button to clear the current selection no matter which tool is selected
- Increased brush smoothing

**Reply by RadTechDad** [reply]
Awesome! Totally understandable about the road map. Thanks for mentioning what is planned for the next release. I'll make sure to put my suggestions in the "ideas" category.
